settings.gradle

```diff
@@ -1,2 +1,6 @@
+plugins {
```

Comment:
What does this plugin do? Maybe add a comment

Reply:
Ah this is the default toolchain resolver of gradle see [the docs](https://docs.gradle.org/current/userguide/toolchains.html#sub:download_repositories). Will add comment if you'd like